pgrx-sql-entity-graph/src/fmt.rs

```diff
@@ -1,3 +1,33 @@
+macro_rules! lazy_err {
+    ($span_expr:expr, $lit:literal $(, $tokens:tt),*) => {
+        {
+            let spanning = $span_expr;
```

Comment:
This is so that the expr can resolve instead of being captured into the closure. This make the name `lazy_err!` slightly confusing, perhaps, since this part is eagerly-evaluated, but a lot of exprs die instead.

Reply:
Fixed by adding docs.